commands/cli.go

```diff
@@ -152,6 +157,11 @@ func CreateCli(version string) *cli.App {
 					Usage: "Only inspect resources older than this specified value. Can be any valid Go duration, such as 10m or 8h.",
 					Value: "0s",
 				},
+				&cli.StringFlag{
```

Comment:
This looks like a duplicate - should it be `older-than`?

Reply:
One is for the `aws` command and another one for `inspect-aws` comamnd.